tasks/transform_conformance/src/lib.rs

```diff
@@ -156,13 +206,14 @@ impl TestRunner {
 struct TestCase {
     path: PathBuf,
     options: BabelOptions,
+    kind: TestCaseKind,
```

Comment:
Do you think having a `ConformanceTestCase` and `ExecTestCase` and abstract the methods with a trait will make the code easier to expand later?